Makefile

```diff
@@ -129,6 +136,10 @@ push-multi-arch-eserver:
 	@echo "Build and $(DOCKER_TARGET) eserver image $(ESERVER_TAG):$(ESERVER_VERSION)"
 	@docker buildx build --$(DOCKER_TARGET) --platform $(DOCKER_PLATFORM) --tag $(ESERVER_TAG):$(ESERVER_VERSION) $(ESERVER_DIR)
 
+push-multi-arch-swtpm:
+	@echo "Build and $(DOCKER_TARGET) swtpm image"
+	@docker buildx build --$(DOCKER_TARGET) --platform $(DOCKER_PLATFORM) --tag $(SWTPM_TAG):$(SWTPM_VERSION) $(SWTPM_DIR)
```

Comment:
Is there any particular reason to use buildx as opposed to normal docker build?

Reply:
Actually no, thanks! I'll revert to plain docker build for the next version. However, after @giggsoff comment, I'm unsure whether swtpm needs to be part of eden, or if we simply should assume that the user has a working swtpm instance (analogous to how eden requires that the user installs qemu). Any thoughts on this?

Reply:
Are you planning to make these changes soon @RobertBuhren ?